editor/src/clj/editor/app_view.clj

```diff
@@ -1178,17 +1178,25 @@
               (fn process-pre-build-hook-results-on-ui-thread! [extension-error]
                 (if (some? extension-error)
                   (finish-with-result! {:error extension-error})
-                  (phase-2-start-all-build-processes!))))))]
+                  (phase-3-start-all-build-processes!))))))
```

Comment:
All changes above are plain renames of phases because I introduced a new phase at the start.